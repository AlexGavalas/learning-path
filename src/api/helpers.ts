import axios from 'axios';

export const fetchFileFromStorage = async (
    filename: string,
): Promise<string> => {
    const fileServerUrl = String(process.env.PUBLIC_FILE_SERVER_URL);

    const response = await axios.get(`${fileServerUrl}/${filename}`, {
        responseType: 'text',
    });

    return response.data as string;
};
