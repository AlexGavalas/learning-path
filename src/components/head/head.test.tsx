import { render } from '~test/helpers';

import { Head } from './head';

describe('<Head />', () => {
    it('renders', () => {
        const { container } = render(<Head />);

        expect(container).toMatchSnapshot();
    });
});
