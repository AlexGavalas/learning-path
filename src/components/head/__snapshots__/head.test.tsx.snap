// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Head /> renders 1`] = `
<div>
  <link
    href="/favicons/favicon-32x32.png"
    rel="icon"
    sizes="any"
  />
  <link
    href="/favicons/favicon.svg"
    rel="icon"
    type="image/svg+xml"
  />
  <link
    href="/favicons/apple-touch-icon.png"
    rel="apple-touch-icon"
    sizes="180x180"
  />
  <link
    href="/site.webmanifest"
    rel="manifest"
  />
  <meta
    content="Learn stuff about software development"
    name="description"
  />
  <meta
    content="Learning Path"
    property="og:site_name"
  />
  <meta
    content="Learning Path"
    property="og:title"
  />
  <meta
    content="Learn stuff about software development"
    property="og:description"
  />
  <meta
    content="/images/spash.jpg"
    property="og:image"
  />
  <meta
    content="https://learning-path.dev/"
    property="og:url"
  />
  <meta
    content="summary_large_image"
    property="twitter:card"
  />
  <meta
    content="https://learning-path.dev/"
    property="twitter:url"
  />
  <meta
    content="Learning Path"
    property="twitter:title"
  />
  <meta
    content="Learn stuff about software development"
    property="twitter:description"
  />
  <meta
    content="/images/spash.jpg"
    property="twitter:image"
  />
  <meta
    content="#121212"
    name="theme-color"
  />
  <link
    href="https://learning-path.dev"
    rel="canonical"
  />
</div>
`;
